Shared/mods/deathmatch/logic/CLatentSendQueue.cpp

```diff
@@ -230,7 +230,7 @@ bool CLatentSendQueue::GetSendStatus ( SSendHandle handle, SSendStatus* pOutSend
             pOutSendStatus->iStartTimeMsOffset = iTimeMsBefore - iter->iEstSendDurationMsUsed;
             pOutSendStatus->iEndTimeMsOffset = iTimeMsBefore + iter->iEstSendDurationMsRemaining;
             pOutSendStatus->iTotalSize = iter->bufferRef->GetSize ();
-            pOutSendStatus->iPercentComplete = iter->uiReadPosition * 100 / std::max ( 1, pOutSendStatus->iTotalSize );
+            pOutSendStatus->dPercentComplete = iter->uiReadPosition * 100 / std::max ( 1, pOutSendStatus->iTotalSize );

```

Comment:
Something on the left side has to be a double, otherwise the result will be still be rounded.